<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="312" started-at="2017-12-30T17:39:41Z" finished-at="2017-12-30T17:39:41Z">
    <groups>
      <group name="Regression">
        <method signature="TestNGSessionTwo.testcase011()[pri:1, instance:TestNGSession.TestNGSessionTwo@18be83e4]" name="testcase011" class="TestNGSession.TestNGSessionTwo"/>
        <method signature="TestNGSessionTwo.testcase012()[pri:2, instance:TestNGSession.TestNGSessionTwo@18be83e4]" name="testcase012" class="TestNGSession.TestNGSessionTwo"/>
        <method signature="TestNGSessionTwo.testcase013(java.lang.String, java.lang.String)[pri:3, instance:TestNGSession.TestNGSessionTwo@18be83e4]" name="testcase013" class="TestNGSession.TestNGSessionTwo"/>
      </group> <!-- Regression -->
    </groups>
    <test name="Test" duration-ms="312" started-at="2017-12-30T17:39:41Z" finished-at="2017-12-30T17:39:41Z">
      <class name="TestNGSession.TestNGSessionTwo">
        <test-method status="FAIL" signature="testcase011()[pri:1, instance:TestNGSession.TestNGSessionTwo@18be83e4]" name="testcase011" duration-ms="109" started-at="2017-12-30T12:39:41Z" description="Login Testcase" finished-at="2017-12-30T12:39:41Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at TestNGSession.TestNGSessionTwo.testcase011(TestNGSessionTwo.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase011 -->
        <test-method status="PASS" signature="testcase013(java.lang.String, java.lang.String)[pri:3, instance:TestNGSession.TestNGSessionTwo@18be83e4]" name="testcase013" duration-ms="16" started-at="2017-12-30T12:39:41Z" description="Testcase Parameter" finished-at="2017-12-30T12:39:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ASVG]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[15]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase013 -->
        <test-method status="SKIP" signature="testcase012()[pri:2, instance:TestNGSession.TestNGSessionTwo@18be83e4]" name="testcase012" duration-ms="0" started-at="2017-12-30T12:39:41Z" description="Logout Testcase" depends-on-methods="TestNGSession.TestNGSessionTwo.testcase011" finished-at="2017-12-30T12:39:41Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestNGSessionTwo.testcase012()[pri:2, instance:TestNGSession.TestNGSessionTwo@18be83e4] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestNGSessionTwo.testcase012()[pri:2, instance:TestNGSession.TestNGSessionTwo@18be83e4] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase012 -->
      </class> <!-- TestNGSession.TestNGSessionTwo -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
